docs/tutorials/Error-Suppression-and-Error-Mitigation.ipynb

```diff
@@ -278,7 +278,7 @@
     "    for _ in range(i):\n",
     "        trotter_circuit = trotter_circuit.compose(trotter_layer)\n",
     "    trotter_circuit_list.append(trotter_circuit)\n",
-    "    print(f'Trotter circuit with {i} Trotter steps`)\n",
+    "    print(\"Trotter circuit with {} Trotter steps\".format(i))\n",
```

Comment:
```suggestion
    "    print(f\"Trotter circuit with {i} Trotter steps\")\n",
```

I think there is a different styling issue in this file but if you run `make black` it should also be resolved